src/client/components/MainHeader/ProfileName.tsx

```diff
@@ -5,74 +5,34 @@ import type { KVKData } from '../../../server/services/kvk';
 import { AppRoutes, FeatureToggle } from '../../../universal/config';
 import { getFullName } from '../../../universal/helpers';
 import { BRPData } from '../../../universal/types';
-import { IconHomeCommercial, IconProfile } from '../../assets/icons';
+import { IconProfile } from '../../assets/icons';
 import { useAppStateReady } from '../../hooks';
-import { Button } from '../Button/Button';
 import LoadingContent from '../LoadingContent/LoadingContent';
 import styles from './ProfileName.module.scss';
+import { PageMenu } from '@amsterdam/design-system-react';
 
 interface CommercialProfileNameProps {
   company?: KVKData;
-  onClick?: (event: any) => void;
-  isActive: boolean;
+  href: string;
 }
 
-function CommercialProfileName({
-  company,
-  onClick,
-  isActive,
-}: CommercialProfileNameProps) {
+function CommercialProfileName({ company, href }: CommercialProfileNameProps) {
   const label = company?.onderneming.handelsnaam || 'Mijn onderneming';
-  return (
-    <Button
-      onClick={onClick}
-      icon={IconHomeCommercial}
-      variant="plain"
-      lean={true}
-      iconSize="24"
-      className={classnames(
-        styles.ProfileLink,
-        styles['ProfileLink--commercial'],
-        isActive && styles['ProfileLink--active']
-      )}
-    >
-      {label}
-    </Button>
-  );
+  return <PageMenu.Link href={href}>{label}</PageMenu.Link>;
```

Comment:
Dit werkt niet in SPA. We hebben een component: `MARouterLink` misschien kun je die gebruiken? Anders een onClick handler.